languages/__init__.py

```diff
@@ -12,17 +13,26 @@
 
 
 def execute(config, database, message, metadata: dict, data: io.BufferedReader):
-    lang = detect_language(metadata, data)
+    try:
+        lang = detect_language(metadata, data)
+        if not lang:
+            log.info("Could not detect language")
+            return
+        save_to_db(database, message, lang)
+    except LangDetectException:
+        log.info("Could not detect language")
 
-    save_to_db(database, message, lang)
 
 
 def detect_language(metadata, data):
     log.info('Detecting language of object')
     lang = metadata.get('lang', None) or metadata.get('language', None)
 
+    MAX_BYTE_SIZE_TO_READ = 10000
+
     if not lang:
-        lang = detect(data.read().decode())
+        stringData = data.read(MAX_BYTE_SIZE_TO_READ).decode()
```

Comment:
Please use naming convention according to [`PEP8` - Function and Variable Names](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)